samples/CustomMiddleware/ExceptionHandlingMiddleware.cs

```diff
@@ -36,8 +38,10 @@ public async Task Invoke(FunctionContext context, FunctionExecutionDelegate next
 
                 if (httpReqData != null)
                 {
-                    var newHttpResponse = httpReqData.CreateResponse();
-                    await newHttpResponse.WriteAsJsonAsync(new { FooStatus = "Invocation failed!" });
+                    // Create an instance of HttpResponseData with 500 status code.
+                    var newHttpResponse = httpReqData.CreateResponse(HttpStatusCode.InternalServerError);
+                    // You need to explicitly pass the status code in WriteAsJsonAsync method due to #776
```

Comment:
What does "#776" mean here?

Reply:
Ah nevermind I guess github just told me :)

Reply:
Agreed. Let's have the full issue URL here.

Reply:
Switched to full issue link.